libs/ngx-mime/src/lib/viewer/viewer.component.html

```diff
@@ -15,18 +15,23 @@
     #mimeOsdToolbar
   ></mime-osd-toolbar>
 
-  <mat-drawer-container class="viewer-drawer-container">
+  <mat-drawer-container class="viewer-drawer-container" autosize>
     <mat-drawer
+      data-test-id="ngx-mimeRecognizedTextContentContainer"
       mode="side"
       position="end"
       (openedChange)="goToHomeZoom()"
-      [opened]="isRecognizedTextContentToggled"
-      [ngClass]="{ open: showHeaderAndFooterState === 'show' }"
-      ><mime-recognized-text-content
-        *ngIf="isRecognizedTextContentToggled"
-      ></mime-recognized-text-content
+      [opened]="recognizedTextContentMode !== recognizedTextMode.NONE"
+      [ngClass]="{
+        only: recognizedTextContentMode === recognizedTextMode.ONLY,
+        split: recognizedTextContentMode === recognizedTextMode.SPLIT,
+        open: showHeaderAndFooterState === 'show'
+      }"
+      ><mime-recognized-text-content></mime-recognized-text-content
     ></mat-drawer>
-    <mat-drawer-content><div id="openseadragon"></div></mat-drawer-content>
+    <mat-drawer-content
+      ><div id="openseadragon" #openseadragon></div
+    ></mat-drawer-content>
```

Comment:
Force prettier to do better formatting.
```suggestion
    <mat-drawer-content>
      <div id="openseadragon" #openseadragon></div>
    </mat-drawer-content>
```